docs/nodejs/specifications.md

```diff
@@ -0,0 +1,71 @@
+---
+order: 100
+---
+# Specifications & Limitations
+
+## Availability
+
+Node.js is an add-on to your existing Altis platform subscription and may come with additional charges. Contact your account manager for more information about adding Node.js to your subscription.
+
+
+## Runtime Environment
+
+Your application runs inside a Docker container, built as part of the build process. This container is based on the [official `node:<version>-slim` container](https://hub.docker.com/_/node) published by Docker.
+
+The following versions are available, and the specific version in use will depend on your configuration.
+
+* Node.js version: 18, 20, 21
+
+**Note:** The operating system environment is not part of the stable API, and is subject to change. We strongly recommend against using Node add-ons (native dependencies) built with node-gyp.
+
+
+### Environment Variables
+
+The following variables are available in the runtime environment:
+
+* `ALTIS_ENVIRONMENT_NAME` - The full ID for the environment (e.g. `yourproject-dev-01`)
+* `ALTIS_ENVIRONMENT_REGION` - The AWS region code for the environment
+* `ALTIS_ENVIRONMENT_TYPE` - The environment type; typically one of `production`, `staging`, `development`. (Add-on environments may use custom types.)
+
```

Comment:
@rmccue The Notion document mentions the `ALTIS_BACKEND_HOST` environment variable. You've not included that here. Is that an oversite, or shall I document that here too?